src/fetch-docs.py

```diff
@@ -22,7 +22,6 @@
     f'{augur_url}faq/fasta_input.md': 'guides/bioinformatics/fasta_input.md',
     f'{augur_url}faq/seq_traits.md': 'guides/bioinformatics/seq_traits.md',
     f'{augur_url}examples/examples.rst': 'guides/bioinformatics/examples.rst',
-    f'{augur_url}usage/cli/snippets/filtering-and-subsampling.rst': 'snippets/filtering-and-subsampling.rst',
```

Comment:
If this PR goes through, I would think to do something similar for all the other fetched docs. They aren't even used in the subprojects ([Augur](https://github.com/nextstrain/augur/blob/430a9766b551ae3c581450bc614bf2250e3d6c05/docs/conf.py#L79-L86), [Auspice](https://github.com/nextstrain/auspice/blob/27509249c40e6bb9691dbe4ff4055052f3dfc68b/docs/conf.py#L46)). This would remove the need for any docs fetching.

Reply:
(From memory) we preferred to co-locate documentation with code (i.e. auspice docs in auspice repo) even tho the decision was made to render the docs in a different RTD subproject.

From a technical standpoint the fetching of docs has a whole bunch of papercuts and I'd be very happy to see us move past this. However for my money it is a _symptom_ of our siloed docs setup, and there are larger organizational / design issues. 

The whole setup of the docs is changeable from my point of view. This is all to say that if you would like to make bigger changes then I'd encourage this.

Reply:
I agree with @jameshadfield that there's still a good bit of holistic work to be done to improve our doc organization. I'm not sure if we agree on what that work is though! James, would you say more (here, or elsewhere) on "our siloed docs setup" and what "larger organizational / design issues" there are to fix in your eyes?

Reply:
Thanks, I agree there's a lot more bigger picture thinking to be done here. Let's continue this in #203